libs/math/include/math/TVecHelpers.h

```diff
@@ -142,6 +158,14 @@ class TVecAddOperators {
     VECTOR<arithmetic_result_t<T, U>> MATH_PURE operator-(const VECTOR<T>& lv, U rv) {
         return lv - VECTOR<U>(rv);
     }
+    friend inline constexpr VECTOR<T> MATH_PURE operator +(VECTOR<T> lv, T s) {
```

Comment:
same comment as above, this should be entirely handled by the functions above that take a T and U.

Reply:
removed.